package.json

```diff
@@ -8,6 +8,7 @@
     "lint:fix": "lage lint:fix -- --color",
     "change": "beachball change",
     "clean": "lage clean -- --color",
+    "doc": "doxysaurus --config vnext/doxysaurus.json",
```

Comment:
If this is just for `vnext`, could you instead add this script to the package.json in `vnext`, then replace this bit with `lage doc -- --color`?

Most of the commands at the root of the repo just run commands on other packages in dependency order.

Reply:
I tried it, but it gives me different behavior from the current one: no logs on screen and cannot pass extra parameters